### ISSUE TITLE
No way of debugging node server

### ISSUE THREAD

**Issue by d4rky-pl** [jaredpalmer/razzle#324]
Running `node --inspect node_modules/razzle/scripts/start.js` causes the server-side reloader to get stuck on the first `debugger` statement but the debugger itself never reacts and none of the server-side files are available to open and add breakpoints.

**Comment by jaredpalmer** [comment]
Cc @ericclemmons how do you inspect a hot server?

**Comment by ericclemmons** [comment]
I haven't.  But I know people have via `start-server-webpack-plugin`:
> https://github.com/ericclemmons/start-server-webpack-plugin#usage

```js
new StartServerPlugin({
      name: 'server.js',
      nodeArgs: ['--inspect'], // allow debugging
      args: ['scriptArgument1', 'scriptArgument2'], // pass args to script
    }),
```

Razzle's `start` script isn't what you want to inspect.  You want to inspect the webpack'd `server.js`.

My recommendation would be something like `razzle start --inspect`.

**Comment by jaredpalmer** [comment]
Shnice. Thanks. Will play with this tonight / tomorrow (at React Rally right now)

**Comment by ericclemmons** [comment]
Have fun! I'm skipping confs unless I'm speaking so...no confs for me :D
          
            
            On Aug 24 2017, at 5:36 pm, Jared Palmer <notifications@github.com> wrote:
            
            
              Shnice. Thanks. Will play with this tonight / tomorrow (at React Rally right now)

**Comment by jariz** [comment]
Fixed w/ #333
